Configuration/TCA/Overrides/sys_file_reference.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+$additionalColumns = [
+    'language' => [
```

Comment:
To be consistent with [TYPO3 naming conventions](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ExtensionArchitecture/BestPractises/NamingConventions.html#database-column-name), this field should be named `tx_html5mediakit_language` or even better `tx_html5mediakit_track_srclang`.